rfcs/shared/build-system/06-browser-support-for-v9.md

```diff
@@ -0,0 +1,61 @@
+# RFC: Browser Support for Fluent v9
+
+Contributors: Fluent UI React Build team, @TristanWatanabe
+
+## Summary
+
+Fluent v9 is currently lacking a defined browser support matrix so this RFC proposes a set of major browser versions compatible with the library to act as the source of truth. As mentioned below, Fluent developers are expected to conform to these browsers and ensure no incompatible syntax, browser APIs or CSS features are introduced to the library. Consumers who require older browser support that aren't covered in this matrix are responsible for performing downlevel transpilations and/or using polyfills themselves.
+
+## Problem Statement
+
+As it stands, Fluent v9 does not currently adhere to a specific browser support matrix. This makes it difficult for partner teams and general consumers of the library to know how using the library will impact their own product. There's also a lack of guidance for the internal team in terms of the syntax, browser APIs and CSS features that they can/can't use.
+
+## Detailed Design or Proposal
+
+### Browser Support Matrix
+
+The proposed browser support matrix will be loosely based off the [Resize Observer](https://caniuse.com/mdn-api_resizeobserver) support matrix . This decision was driven by the current usage of the `ResizeObserver` in Fluent v9 as well as the current business goals of our partner teams. Along with that, being able to maintain our current ES2019 target and eventually move to ES2020 without the need to transpile down was also a factor. As it stands, the lack of adequate polyfill capabilities for CSS features in tandem with the goals of our partners is the biggest detriment to the support matrix not being higher.
+
+| Edge | Firefox | Chrome | Safari | Opera | Safari on IOS | Chrome for Android | Samsung | Internet Explorer |
```

Comment:
wondering about these 2 chrome versions. the version difference is quite big.

![image](https://user-images.githubusercontent.com/1223799/168251166-7b8a08ae-054b-4cab-b5fa-cbeb00f245dd.png)

AFAIR chrome on android is "on par" with desktop v8 core. If that's true this doesn't make much sense and mobile version should be lowered. WDYT?

Reply:
Ah you’re right, they are on par - i’ll update the mobile version to 80 as well